applauncher/004-three-by-three.qml

```diff
@@ -0,0 +1,310 @@
+            forbidRight = false
+        }
+    }
+
+    MouseArea {
+        id: absoluteMouse
+
+        anchors.fill: parent
+        propagateComposedEvents: true
+
+        onPressed: {
+            clickY = Math.round(mouse.y)
+            mouse.accepted = false
+        }
+        onClicked: mouse.accepted = false
+        onReleased: mouse.accepted = false
+        onDoubleClicked: mouse.accepted = false
+        onPositionChanged: mouse.accepted = false
+        onPressAndHold: mouse.accepted = false
+    }
+
+    Rectangle {
+        id: titleShutter
+
+        opacity: 0
+        visible: !root.clickToggle
+        anchors {
+            centerIn: hoverTitle
+            verticalCenterOffset: root.clickY > Dims.h(48) ? -Dims.h(2) : Dims.h(2)
+        }
+        width: root.width
+        height: root.height * .24
+        color: alb.centerColor(launcherModel.get(currentPressedIndex).filePath)
+    }
+
+    Text {
+        id: hoverTitle
+
+        //Offset hoverText and shutter out of view in L290, either top or bottom depending on vertical click position
+        property real hoverTextOffset: root.clickY > Dims.h(48) ? -Dims.h(62) : Dims.h(62)
+        property bool rootPressToggle: root.pressToggle
+
+        width: parent.width
+        color: "#fff"
+        opacity: 0
+        visible: !root.clickToggle
+        horizontalAlignment: Text.AlignHCenter
+        anchors.centerIn: parent
+        style: Text.Outline;
+        styleColor: alb.centerColor(launcherModel.get(currentPressedIndex).filePath)
+        font {
+            pixelSize: ((appsView.width > appsView.height ? appsView.height : appsView.width) / Dims.l(100)) * Dims.l(9)
+            styleName: "Condensed Medium"
+            letterSpacing: -parent.width * .004
+        }
+        text: appTitle.toUpperCase() + localeManager.changesObserver
+
+        Behavior on rootPressToggle {
+            SequentialAnimation {
+                id: fadeText
+
+                //Reset position of all animated items for the case an animation has been interrupted
+                NumberAnimation { target: hoverTitle; property: "anchors.verticalCenterOffset"; to: hoverTitle.hoverTextOffset; duration: 0}
+                NumberAnimation { target: hoverTitle; property: "opacity"; to: 1; duration: 0}
+                NumberAnimation { target: titleShutter; property: "opacity"; to: .85; duration: 0}
+
+                PropertyAction { }
+
+                //Slide in hoverTitle and shutter to either negative or positve offset from center depending on vertical click position
+                NumberAnimation { target: hoverTitle; property: "anchors.verticalCenterOffset"; to: -hoverTitle.hoverTextOffset + (hoverTitle.hoverTextOffset * 1.58); duration: 100; easing.type: Easing.InSine}
+
+                //Keep hoverTitle in visible position for 1s
+                PauseAnimation { duration: 1000 }
+
+                //Slide hoverTitle and shutter out of view again
```

Comment:
Thanks for the comments, for consistency please keep a space between the `//`s and the comments.